py/tests/plane_move.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+import pykms
+import random
+import time
+import sys
+import select
+import selectors
+
+if len(sys.argv) not in (1, 3):
+    print('Without parameters: list of available connectors')
+    print('{} [name of connector0] [name of connector1]\n'.format(sys.argv[0]))
+    sys.exit(-1)
+
+card = pykms.Card()
+
+if not card.has_atomic:
+    print('Atomic mode settings is not supported :(')
+    sys.exit(-1)
+
+if len(sys.argv) == 1:
+    print('List of available connectors:')
+    for conn in card.connectors:
```

Comment:
This is pointless code, we have kmsprint to show information about DRM objects.

Reply:
True. On the other hand I have found it handy to not need to run kmsprint to get the list of the connectors but give a short list here for the user.

Reply:
Do you even need to run kmsprint? I know we have, say, HDMI and DPI connectors. Or if I want to be lazy, connectors 0 and 1. That's enough to fill in the parameters.
If we add such code and want to be consistent, we'd be adding similar code, or perhaps listing planes or crtcs, to most of our utilities. I mean, "cp" command will not list the files in a directory =).

Reply:
Fair enough.